### ISSUE TITLE
Vote on jupyter/accessibility's status in the new Jupyter governance

### ISSUE THREAD

**Issue by isabela-pf** [jupyter/accessibility#81]
## What we need

We’ve had some discussions about jupyter/accessibility’s place in the new [Jupyter governance](https://jupyter.org/governance/intro.html) on this [jupyter/governance PR](https://github.com/jupyter/governance/pull/121),  [jupyter/governance issue](https://github.com/jupyter/governance/pull/121), and in the last two [JupyterLab accessibility meetings](https://github.com/jupyter/accessibility/blob/master/docs/community/meeting-minutes/jupyterlab-accessibility-meetings/2022-03-23.md). We need to come to an agreement on how we’d like to move forward as a community.

## The story so far

This a quick summary of the discussion so far. Please read the [jupyter/governance issue](https://github.com/jupyter/governance/pull/121) or the [JupyterLab accessibility meeting notes](https://github.com/jupyter/accessibility/blob/master/docs/community/meeting-minutes/jupyterlab-accessibility-meetings/2022-03-23.md) for more in-depth information.
- [jupyter/accessibility was added to the new governance structure as a working group](https://github.com/jupyter/governance/pull/121/files#diff-26acd6341ac14d705b75267e8eb9a168fa509a4e051aa9a386aead5fb2937836R19).
- There was a [counter-proposal that jupyter/accessibility should be a working group and software sub-project](https://github.com/jupyter/governance/issues/124)
- At [a governance call](https://github.com/jupyter/governance/issues/124#issuecomment-1058674231), it was mentioned that jupyter/accessibility has standing to have both a working group and software project standing, they just need separate administration and would need to report separately.
- With any of the above options, jupyter/accessibility will have one representative and vote in the [Software Steering Council](https://github.com/jupyter/governance/pull/121/files#diff-a854648dd6440887bb685363052c82cf6a6ad553c653fe060f3c2858e890d2a3R36). (I have no link for this fact, it has been expressed verbally so far.)

## How to vote

We can go with any of these options, so I’m proposing a vote by reacting to this comment.
- react with 🎉  if you are in favor of jupyter/accessibility to join as a [working group](https://jupyter.org/governance/standing_committees_and_working_groups.html).
- react with 🚀  if you are in favor of jupyter/accessibility to join as a [software sub-project](https://jupyter.org/governance/software_subprojects.html).
- react with ❤️  if you are in favor of jupyter/accessibility to join as both a working group and software sub-project.
- react with 👎  if you are in favor of none of these options (I’d appreciate a follow-up comment for this vote since no one has presented this argument yet and it’d be good to document why).
- react with 😕 to abstain from voting.

Tagging people that have been at the meetings we’ve discussed this (who left a github handle)…
@krassowski, @ohrely, @marthacryan @gabalafou @trallard @tonyfast @fcollonval, @bollwyvl, @jasongrout, @afshin

As well as @jupyter/accessibility in general.

Please discuss on this issue as needed, but reserve reacting to this comment for voting! Thank you!

Voting will end midnight Wednesday, April 20 [anywhere on Earth](https://www.timeanddate.com/time/zones/aoe). (If this doesn’t work for any reason, let me know and we can reschedule).

**Comment by fcollonval** [comment]
I would not create both a working group and a sub-project for the same subject. If there is two bodies, this means we can draw a boundary between the two and we need decision bodies for the two bodies.

So sticking with only one body will be clearer. As a _working group_ is not expected to be permanent:

> The primary difference between Standing Committees and Working Groups is that Standing Committees are intended to be permanent

It makes sense to consider accessibility as a sub-project as it should be a permanent concern for current and future Jupyter projects developments.

**Comment by isabela-pf** [comment]
Reminder to all tagged on this that midnight (wherever you are) is the deadline for voting on this. Thank you!

**Comment by isabela-pf** [comment]
Okay, based on the votes we will be moving forward with jupyter/accessibility joining as a [software sub-project](https://jupyter.org/governance/software_subprojects.html). Thank you all for voting.

I will keep this issue open for updates until that change is made over in [jupyter/governance](https://github.com/jupyter/governance/). Please feel free to ask any questions.

**Comment by trallard** [comment]
Thanks for taking care of this @isabela-pf  💜

**Comment by isabela-pf** [comment]
I'm closing this because the status has changed. Jupyter/accessibility is now a subproject! We will be organizing the next steps needed for this membership in [JupyterLab accessibility calls](https://github.com/jupyter/accessibility#handshake-join-our-jupyterlab-accessibility-meetings) and posting in issues here to keep everyone up to date. The [notes for these meetings](https://github.com/jupyter/accessibility/tree/main/docs/community/meeting-minutes/jupyterlab-accessibility-meetings) are another place to find updates as they are posted.

Thank you all for participating in our community and this vote! I'm grateful to be here with you all! 🌻
